infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -332,12 +325,11 @@ private static void clearDataFiles(ServerContext serverContext) {
      */
     static void restartServer(boolean resetData) {
 
-        if (resetData) {
-            cleanupServer = true;
-        }
-
         log.info("RestartServer: Shutting down corfu server");
         activeServer.close();
+        if (resetData) {
```

Comment:
Can you change this variable name. Reset and restart have been confusing to many.

Reply:
fixed